ivadomed/scripts/automate_training.py

```diff
@@ -138,24 +138,24 @@ def automate_training(fname_config, fname_param, fixed_split, all_combinations,
     # TODO: add example of DF
 
     Args:
-        fname_config (string): Configuration filename, which is used as skeleton to configure the training. Some of its
-            parameters (defined in `fname_param` file) are modified across experiments.
-        fname_param (string): json file containing parameters configurations to compare. Parameter "keys" of this file
-            need to match the parameter "keys" of `fname_config` file. Parameter "values" are in a list. Example::
+        config (string): Configuration filename, which is used as skeleton to configure the training. Some of its
+            parameters (defined in `param` file) are modified across experiments.
+        param (string): json file containing parameters configurations to compare. Parameter "keys" of this file
+            need to match the parameter "keys" of `config` file. Parameter "values" are in a list. Example::
 
                 "default_model": {"depth": [2, 3, 4]}
 
         fixed_split (bool): If True, all the experiments are run on the same training/validation/testing subdatasets.
-        all_combinations (bool): If True, all parameters combinations are run.
+        all_combin (bool): If True, all parameters combinations are run.
```

Comment:
i would avoid mixing _ and -, so i would go for `--all-combin` instead

and i don't see any problem in *not* having a single letter (e.g. `-a`) if none that makes sense is available

Reply:
>i would avoid mixing _ and -, so i would go for --all-combin instead

we can use that as the flag, but the parameter will still be all_combin. (python won't accept all-combin as this can be read as 'all minus combin'. 

>On the RTD, the flags are not indicated (-c or --config corresponds to the config parameter). I know it is implicit with the parameter names: first letter of the parameter is the flag, but it is not always the case.

I will add that in the doc.

Reply:
Done in cd640d6